packages/framework-integration-tests/integration/end-to-end/cart.integration.ts

```diff
@@ -1,438 +0,0 @@
-import { ApolloClient } from 'apollo-client'
```

Comment:
I divided this file into three files that test different components:
- Commands
- Entities
- ReadModels
- Events

The only one where I added tests was in the events one.